tools/topology/topology2/include/components/dai-copier.conf

```diff
@@ -137,6 +137,9 @@ Class.Widget."dai-copier" {
 
```

Comment:
"These are" or "They are"

Reply:
fixed and now the kernel dependency is merged as well. So lets retest